package.json

```diff
@@ -39,7 +41,7 @@
     "node": ">= 18"
   },
   "scripts": {
-    "lint": "eslint --plugin markdown --ext js,md .",
+    "lint": "eslint --flag unstable_config_lookup_from_file .",
```

Comment:
Only way I found to add the needed config for tests. This flag enables the [new lookup behaviour](https://eslint.org/docs/latest/use/configure/configuration-files#experimental-configuration-file-resolution) starting in eslint v10.0.0,